static/index.html

```diff
@@ -105,7 +105,7 @@ <h3>Drupal in Cebu</h3>
                 <h3>Drupal on Twitter</h3>
                 <p> <i class="fab fa-twitter"></i><a href="https://twitter.com/DrupalCampCebu" target="_blank">@DrupalCampCebu.</a> Who said conferences are not funny? More photos from our last Drupalcamp Cebu: <a href="https://www.facebook.com/media/set/?set=oa.723538837835150&type=3" target="_blank">https://www.facebook.com/media/set/?set=oa.723538837835150&type=3</a><a href="https://twitter.com/hashtag/drupalcampcebu?src=hash" target="_blank"> #drupalcampcebu</a></p>
```

Comment:
<img width="305" alt="screen shot 2018-04-05 at 10 53 27 am" src="https://user-images.githubusercontent.com/8597102/38356345-c74951ca-38bf-11e8-9127-4266699ed264.png"> 
I think the link here will not make sense to our end-users. We can make it more user-friendly by providing a more comprehensible text e.g. 'Official photos from last year's Drupalcamp Cebu 2017'. :)